docs/user_manual.md

```diff
@@ -502,6 +507,34 @@ You can build this kind of configuration easily from the sample deployment by ch
 * Change the Roles to ClusterRoles
 * Change the RoleBindings to ClusterRoleBindings
 
+# Choosing Your Public IP Source
+
+The default behavior of the operator is to use the IP assigned to the pod as the public IP for FoundationDB. This is not the right choice for some environments, so you may need to consider an alternative approach.
+
+## Pod IPs
+
+You can choose this option by setting `spec.services.publicIPSource=pod`. This is currently the default selection.
+
+In this mode, we use the pod's IP as both the listen address and the public address. We will not create any services for the pods.
+
+Using pod IPs can present several challenges:
+
+* Deleting and recreating a pod will lead to the IP changing. If the process is a coordinator, this can only be recovered by changing coordinators, which requires that a majority of the old coordinators still be functioning on their original IP.
+* Pod IPs may not be routable from outside the Kubernetes cluster.
+* Pods that are failing to schedule will not have IP addresses. This prevents us from excluding them, requiring manual safety checks before removing the pod.
+
+## Service IPs
+
+You can choose this option by setting `spec.services.publicIPSource=service`. This feature is new, and still experimental, but we plan to make it the default in the future.
+
+In this mode, we create one service for each pod, and use that service's IP as the public IP for the pod. The pod IP will still be used as the listen address. This ensures that IPs stay fixed even when pods get rescheduled, which reduces the need for changing coordinators and protects against some unrecoverable failure modes.
+
+Using service IPs presents its own challenges:
+
+* In some networking configurations, pods may not be able to access service IPs that route to the pod. See the section on hairpin mode in the [Kubernetes Docs](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-service/#a-pod-fails-to-reach-itself-via-the-service-ip) for more information.
+* Creating one service for each pod may cause performance problems for the Kubernetes cluster
+* We currently only support services with the ClusterIP type. These IPs may not be routable from outside the Kubernetes cluster.
```

Comment:
I would also add that the Service CIDR is normally not that big compared to the Pod CIDR which means it could happen that the cluster runs out of Service IPs.